src/gpuarray/abi_version.h

```diff
@@ -0,0 +1,3 @@
+#ifndef GPUARRAY_ABI_VERSION
+#define GPUARRAY_ABI_VERSION "1.0"
+#endif
```

Comment:
I think this file should not be commited. Add it in the .gitignore file to help prevent readding it.